packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -366,7 +389,7 @@ export class GPUPicker {
         // Cleaning up
         this._pickingTexure?.dispose();
         this._pickingTexure = null;
-        this._renderMaterial?.dispose();
-        this._renderMaterial = null;
+        this._defaultRenderMaterial?.dispose();

```

Comment:
let's make sure to also wipe the map and we are all good!

Reply:
Good point, I'll do that tomorrow

Reply:
Looking at the code again, that already happens with the `this.setPickingList(null);` call. Or would you prefer it being set to `null`?

Reply:
Correct! we are all good :)